### ISSUE TITLE
Options for low-spec hardware? How to set internal render resolution?

### ISSUE THREAD

**Issue by wn2000** [flyinghead/flycast#140]
Platform / OS / Hardware: arm64 / Linux / RK3328 SoC, SDL2 and GLES2

This is a question rather than a bug report. Apology if it's not supposed to be put here.
In the libretro version of flycast, there is an option to set internal rendering resolution. How can I set the same in the standalone version?
I tried to set size in the `[window]` section but that does not seem to change internal resolution.

Basically, I'm trying to run this on a RK3328/Mali-450 aarch64 Linux platform. What are the options I can set to boost the performance? I've tried disabling clipping and fog and that does improve the FPS. Just wondering what else I can do. Using the libretro version if I set a lower internal resolution than the native 640x480, it has an significant effect on FPS.

Appreciate any suggestions.

**Comment by flyinghead** [comment]
You can use the Video > Scaling option to reduce internal resolution. Setting it to 0.5 will reduce the internal resolution by half for example. So if your screen is 1920x1080, internal resolution will be set to 960x540.

**Comment by wn2000** [comment]
Thanks @flyinghead . Did not realize it's based on screen size. So if the screen is 16x9 but the game is 4x3, does the internal resolution take the scaled height and use aspect ratio to figure out the width?
My screen is 1280x720 so I set it to 0.3 to get something closer to 320x240. Now it runs at about 40-50 FPS which is similar to what I got using the libretro core. But the sound is a lot choppier than the libretro version. Anything I can do to improve that?
Also, I've seen Mali-450 mentioned by other users. Is it really that bad in running this?
Last question, I don't see Clipping/Shadow/Fog switches in the libretro version, is it possible to add that?
Or should I submit a request on that repository instead?
Lots of questions :-) Thanks again for your help.

**Comment by flyinghead** [comment]
If supported by the audio driver you use, you can increase the Audio Latency to have a larger audio buffer.
You can also try different audio drivers, including some that aren't compiled by default.
Requests for the libretro version should be done in the libretro repo
